tests/e2e/commands/upgrade.go

```diff
@@ -163,3 +163,24 @@ func ApplyUpgradeLocal(subnetName string) (string, error) {
 	}
 	return string(output), err
 }
+
+func ApplyUpgradePublic(subnetName, avagoChainConfDir string) (string, error) {
```

Comment:
should be called ApplyNetwokUpgradePublic so it is clear which king of upgrade are we talking about

Reply:
Hmm not so sure about this. We are upgrading a single running node when doing public. Calling this Network is kinda misleading, we are not upgrading a network but a node. Maybe it should be called ApplyUpgradeToPublicNode?

Reply:
I mean this is not a vm upgrade, but a network upgrade, or the way it needs to be called